src/payment/strategies/stripev3/stripev3-initialize-options.ts

```diff
@@ -52,4 +54,6 @@ export default interface StripeV3PaymentInitializeOptions {
     containerId: string;
 
     options?: StripeElementOptions | IndividualCardElementOptions;
+
+    form?: HostedFormOptions;
```

Comment:
If you only mean to pass the two fields i.e. card_number and cvv, then please can we use that interface, since `HostedFormOptions` is an interface that is much more than that and can cause confusion.

Reply:
HI @animesh1987, I decided to switch approach and use Stripe components to confirm card number and cvv.

Reply:
Is this still needed here?

Reply:
Yes, it is needed. In fact `fields` field is mandatory only and the rest are optional.